dbms/tests/queries/0_stateless/00578_merge_table_and_table_virtual_column.reference

```diff
@@ -3,6 +3,3 @@
 1000
 1000
 1000
-0
```

Comment:
Changes to test behavior. zero should be different from `non_existing`. But I'm not sure.

Reply:
It's important to heave zeroes here.